src/Raven.Server/RavenServer.cs

```diff
@@ -1653,7 +1654,10 @@ private bool CertHasKnownIssuer(X509Certificate2 userCertificate, X509Certificat
             X509Certificate2 issuerCertificate = null;
 
             var userChain = new X509Chain();
+            userChain.ChainPolicy.DisableCertificateDownloads = true;
+
             var knownCertChain = new X509Chain();
+            knownCertChain.ChainPolicy.DisableCertificateDownloads = true;
```

Comment:
This is the one place that I'm not certain what we should do.
Here we _need_ to insect the whole chain. This is also likely to be relatively rare.
But we need to add a comment explaining why not.